tests/test_importers/test_xsd_importer.py

```diff
@@ -0,0 +1,59 @@
+from linkml_runtime import SchemaView
+from schema_automator.importers import XsdImportEngine
+import tempfile
+
+def parse_string(xsd: str) -> SchemaView:
+    engine = XsdImportEngine()
+    with tempfile.NamedTemporaryFile(mode='w', delete=False) as f:
+        f.write('''<xsd:schema
+            xmlns="http://www.dummy/name/space"
+            targetNamespace="http://www.dummy/name/space"
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            xmlns:xml="http://www.w3.org/XML/1998/namespace"
+            version="2"
+            elementFormDefault="qualified">
+        ''')
+        f.write(xsd)
+        f.write("</xsd:schema>")
+    schema = engine.convert(f.name)
+    return SchemaView(schema)
+
+def test_embedded_type():
+    schema = parse_string('<xsd:element name="AcquisitionDate" type="xsd:dateTime"/>')
+    assert len(schema.all_classes()) == 1
+    root = schema.get_class("SchemaRoot")
+    assert len(root.attributes) == 1
+    assert root.attributes["acquisitionDate"].range == "DateTime"
+    assert "xsd:element" in root.attributes["acquisitionDate"].instantiates
+    assert "xsd" in schema.schema.prefixes
+
```

Comment:
Test coverage of xsd_import_engine.py is 52%. The tests that are here are clear. Some of that is unimplemented cases that I'm not too concerned about, but some of it seems to be relevant. I'm going to see about finding some additional examples of my own.